src/signalcorr.jl

```diff
@@ -403,15 +469,24 @@ function pacf_yulewalker!{T<:RealFP}(r::RealMatrix, X::Matrix{T}, lags::IntegerV
     end
 end
 
+
+"""
+    pacf!(r, X, lags[, method=:regression])
+
+Compute the partial autocorrelation function (PACF) of a matrix `X` at `lags` and
+store the result in `r`. The function accepts an optional keyword argument `method`
+that designates the estimation method. Recognized values are `:regression` (default)
+and `:yulewalker`.
+"""
 function pacf!{T<:RealFP}(r::RealMatrix, X::Matrix{T}, lags::IntegerVector; method::Symbol=:regression)
     lx = size(X, 1)
     m = length(lags)
     minlag, maxlag = extrema(lags)
     (0 <= minlag && 2maxlag < lx) || error("Invalid lag value.")
-    size(r) == (m, size(X,2)) || raise_dimerror()
+    size(r) == (m, size(X,2)) || throw(DimensionMismatch())
```

Comment:
@andreasnoack Do you think this is OK, or was `raise_dimerror()` used to avoid a GC root in performance-sensitive functions?